src/allotropy/parsers/beckman_vi_cell_xr/vi_cell_xr_reader.py

```diff
@@ -78,19 +79,22 @@ def _read_data(self) -> pd.DataFrame:
             errors="coerce",
         )
         file_data = file_data.dropna(subset=DATE_HEADER)
-
-        return file_data
+        return [SeriesData(x) for _, x in file_data.iterrows()]
 
     def _get_file_header(self, header_row: int) -> list[str]:
         """Combine the two rows that forms the header."""
-        header = self._read_excel(
-            nrows=2,
-            skiprows=header_row,
-            header=None,
-        ).fillna("")
+        header = (
```

Comment:
Note: this is a re-fixing of a recently fixed bug. The join below could fail if all elements weren't strings. I realized I can cast to string in pd, instead of in the list below